extras/craft/gulprev/twigextensions/GulpRevTwigExtension.php

```diff
@@ -30,12 +30,13 @@ public function gulpRev($file)
 
         // If the file is not defined in the asset manifest
         // just return the original string
-        $path = $file;
+        $path          = $file;
+        $manifest_path = $_SERVER['DOCUMENT_ROOT'] . '/rev-manifest.json';
```

Comment:
@greypants @tommymarshall - this should do the trick and be more reliable with locales and different craft paths.  Look good?